api/v1alpha1/nutanix_clusterconfig_types.go

```diff
@@ -76,3 +77,17 @@ func (s NutanixPrismCentralEndpointSpec) ParseURL() (string, uint16, error) {
 
 	return hostname, uint16(port), nil
 }
+
+func (s NutanixPrismCentralEndpointSpec) ParseIP() (netip.Addr, error) {
+	pcHostname, _, err := s.ParseURL()
+	if err != nil {
+		return netip.Addr{}, err
```

Comment:
nit: returning an empty struct here instead of `nil` kinda smells

Reply:
yeah, looks weird at first.. I just followed `net/netip` pkg semantic, maybe `Addr` struct doesn't hold much data so they return empty struct instead of nil everywhere.